fusor-ember-cli/app/templates/review/progress/overview.hbs

```diff
@@ -49,4 +49,4 @@
 
 {{/if}}
 
-{{partial 'abandon-deployment-modal'}}
+{{abandon-deployment-modal openModal=openModal}}
```

Comment:
@dymurray, thanks for catching this. I just added PR #854 to also add the action `executeAbandonment='executeAbandonment'`  when calling the component